### PR TITLE
Stable20161206

### PR DESCRIPTION
**Pull request by kbrownell**
As with, https://github.com/SU-SWS/Stanford-Drupal-Profile/pull/68, let me know if these versions look correct based on: https://docs.google.com/spreadsheets/d/1gcywuj_LErJtzU9vfbr5g5stq_KrEQAJYO7w7gp7Qgg/edit#gid=2045289217. Same note regarding jw_player applies: Shea requested we use this jw_player commit instead of version 2.0-beta2: https://github.com/SU-SWS/stanford-jumpstart-deployer/blob/7.x-5.x/development/product/jumpstart-engineering/contrib.make#L221.

### CODE REVIEW

**Comment by kbrownell** [comment]
https://help.github.com/articles/dealing-with-line-endings combined with `grep -rl ⏎ development | xargs sed -i '' 's/⏎//g'` appears to have done the trick.

**Comment by jbickar** [comment]
`development` builds all complete successfully

**Comment by jbickar** [comment]
`production` builds all complete successfully.

@sherakama you will want to look at this also.

I'll do a manual review of changes as well.

**Comment by kbrownell** [comment]
Thank you, both for double checking the changes!

File: development/product/jumpstart-plus/contrib.make [hunk 1]
```diff
@@ -171,29 +175,27 @@ projects[views][version] = "3.14"
...
 projects[workbench_moderation][subdir] = "contrib"
-projects[workbench_moderation][version] = "1.4"
+projects[workbench_moderation][version] = "3.0"
 projects[wysiwyg][subdir] = "contrib"
 projects[wysiwyg][version] = "2.2"
 projects[wysiwyg_filter][subdir] = "contrib"
 projects[wysiwyg_filter][version] = "1.6-rc3"
 projects[xmlsitemap][subdir] = "contrib"
 projects[xmlsitemap][version] = "2.3"
-projects[menu_attributes][subdir] = "contrib"
```
Comment (jbickar) [inline development/product/jumpstart-plus/contrib.make:195]: Yay for alphabetization!

File: production/product/jumpstart/contrib.make [hunk 2]
```diff
@@ -169,19 +173,22 @@ projects[views][version] = "3.14"
...
 projects[webform][subdir] = "contrib"
-projects[webform][version] = "4.13"
+projects[webform][version] = "4.14"
 projects[workbench][subdir] = "contrib"
 projects[workbench][version] = "1.2"
 projects[workbench_access][subdir] = "contrib"
 projects[workbench_access][version] = "1.4"
+projects[workbench_moderation][subdir] = "contrib"
+projects[workbench_moderation][version] = "3.0"
+
```
Comment (jbickar) [inline production/product/jumpstart/contrib.make]: Don't need a blank line here, do we @kbrownell?

**Comment by jbickar** [comment]
Looks good to me. I await @sherakama's green light.

File: production/product/jumpstart/contrib.make (hunk 2 above)
Comment (kbrownell) [reply production/product/jumpstart/contrib.make]: Don't believe we do.  I removed it in: 72810c8.

**Comment by sherakama** [comment]
Code looks good and I am happy to hear that the builds do build. 

Did anyone check on the upgrade paths for these modules? Ie: Does workbench go from 1.0 to 3.0 without breaking? Does it enable the new dependency? 

Example scenario that I am concerned about:
* Site on sites.stanford.edu using workbench 1.0
* Updates upgrade workbench to 3.0 and run updb but drafty is not enabled
* Site WSODs

**Comment by kbrownell** [comment]
Drafty is required by Workbench Moderation (not Workbench).  Testing the upgrade with a drush updb is how we discovered that Drafty was required.  From updb: 

`Workbench_moderation  7300  Enable the Drafty module, a new dependency.`

**Comment by jbickar** [comment]
🎉

**Pull request merged**